params/config_test.go

```diff
@@ -136,3 +142,64 @@ func TestCheckCompatible(t *testing.T) {
 		}
 	}
 }
+
+func TestConfigUnmarshalJSON(t *testing.T) {
+	require := require.New(t)
+
+	testRewardManagerConfig := rewardmanager.NewConfig(
+		big.NewInt(1671542573),
+		[]common.Address{common.HexToAddress("0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC")},
+		nil,
+		&rewardmanager.InitialRewardConfig{
+			AllowFeeRecipients: true,
+		})
+
+	testContractNativeMinterConfig := nativeminter.NewConfig(big.NewInt(0),
+		[]common.Address{common.HexToAddress("0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC")},
+		nil,
+		nil,
+	)
+
+	config := []byte(`{
+    "chainId": 43214,
+    "allowFeeRecipients": true,
+    "rewardManagerConfig": {
+      "blockTimestamp": 1671542573,
+      "adminAddresses": [
+        "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
+      ],
+      "initialRewardConfig": {
+        "allowFeeRecipients": true
+      }
+    },
+    "contractNativeMinterConfig": {
+      "blockTimestamp": 0,
+      "adminAddresses": [
+        "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
+      ]
+    }
+  }`)
```

Comment:
Can we fix the indenting of the JSON here?

Reply:
done